pkg/cli/host_preflight.go

```diff
@@ -137,7 +137,7 @@ func newHostPreflightCmd(cli CLI) *cobra.Command {
 				return errors.Wrap(err, "run host preflight")
 			}
 
-			printPreflightResults(cmd.OutOrStdout(), results)
```

Comment:
why the change to no longer use outorstdout?

I feel like this is the change that's actually making a difference

Reply:
Yep, I agree I reverted that.